packages/kit/types/page.d.ts

```diff
@@ -2,11 +2,12 @@ import { Location as Page, MaybePromise, InferValue } from './helper';
 
 export type LoadInput<
 	PageParams extends Record<string, string> = Record<string, string>,
-	Context extends Record<string, any> = Record<string, any>
+	Context extends Record<string, any> = Record<string, any>,
+	Session = any
```

Comment:
You need to adjust the `Load` and `ErrorLoad` type accordingly

Reply:
@dummdidumm Thanks, will look at that shortly.

Reply:
@dummdidumm updated Load and ErrorLoad 👍